addons/fetchmail/views/res_config_settings_views.xml

```diff
@@ -16,6 +16,10 @@
                     name="%(base.action_ir_mail_server_list)d"
                     string="Outgoing Email Servers" icon="fa-arrow-right" class="btn-link"/>
                 </div>
+                <div icon="fa-arrow-right">
+                    <a href="https://www.odoo.com/documentation/user/11.0/discuss/email_servers.html"><button
```

Comment:
Shouldn't be the documentation of `master`? 🤔

Reply:
Version 11 is latest followed by master.:)